Create ui-c8y-1021-34-2-ui-make-device-info-delete-button-use-lwm-2-m-agent.md (#2865)
SoftwareAG/c8y-docs: content/protocol-integration/lwm2m-bundle/registering-lwm2m-devices.md

### Device deletion {#device-deletion}

> To remove a LWM2M device, delete it through the [All devices](/device-management-application/viewing-all-devices/#to-delete-devices) list in the Device Management application or the [Device Status](/device-management-application/viewing-device-details/#info) card of the device info page.

Reviewer: ```suggestion
To remove a LWM2M device, delete it through the [All devices](/device-management-application/viewing-all-devices/#to-delete-devices) list in the Device Management application or via the **Device status** card on the [**Info** tab](/device-management-application/viewing-device-details/#info) in the device details.
```

Reviewer: Will change it in the 2025 release branch directly as the PR has already been merged
